### DISCUSSION TITLE
TreeTable (with drag drop, search etc)

### DISCUSSION THREAD

**Discussion by pjsaran** [cosscom/coss#54] [Suggestions]
A Tree Table with drag drop to nest, search, custom actions etc. I have a working example but my UI not so great. Attached a sample image and my code.

![image](https://github.com/user-attachments/assets/76c016b3-996c-487b-b513-191ea067903f)
[category-tree.zip](https://github.com/user-attachments/files/17742160/category-tree.zip)

**Comment by jorgedanisc** [comment]
[React Arborist](https://github.com/brimdata/react-arborist) does a great job with this. In case it helps
